sentry/templates/ingress.yaml

```diff
@@ -22,6 +22,20 @@ spec:
             backend:
               serviceName: {{ template "sentry.fullname" . }}-nginx
               servicePort: {{ .Values.nginx.service.port }}
+    {{- else }}
```

Comment:
I think using `else if` would be easier to read: https://helm.sh/docs/chart_template_guide/control_structures/#ifelse

Reply:
okay no prob :)